plugins/modules/ovirt_nic.py

```diff
@@ -71,6 +71,19 @@
         description:
             - Defines if the NIC is linked to the virtual machine.
         type: bool
+    network_filter_parameters:
+        description:
+            - "The list of network filter parameters."
+        elements: dict
+        type: list
+        version_added: 3.1.0
```

Comment:
Do we need 3.1.0 version? Isn't 3.0.0 enough as it hasn't been officially released yet?